### PR TITLE
[AIO] Gamba Suite

### PR DESCRIPTION
**Pull request by JTD420**
A modified version of Nanobytes Poker extension that does all games (_Poker, Tri, 21 & 13_) instead of just poker and tri.

### CODE REVIEW

**Comment by JTD420** [comment]
Sorry for the tag not sure if you get notifs when a PR is submitted? @WiredSpast

**Comment by WiredSpast** [comment]
> Sorry for the tag not sure if you get notifs when a PR is submitted? @WiredSpast

I do but I'm mid exams so don't have a lot of time to review PR's

File: store/extensions/[AIO] Gamba Suite/extension.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
+{
+  "title": "[AIO] Gamba Suite",
+  "description": "Pkr, 13/21 and Tri dice automated rolling and resetting with in-chat hand evaluation. The all-in-one dice management extension.",
+  "authors": [
+    {
+      "name": "JTD",
+      "hotel": ".com",
+      "username": "JTD",
+      "discord": "@JTD420"
```
Comment (WiredSpast) [inline store/extensions/[AIO] Gamba Suite/extension.json]: remove the @ and only lowercase for discord usernames
```suggestion
      "discord": "jtd420"
```

**Comment by JTD420** [comment]
> > Sorry for the tag not sure if you get notifs when a PR is submitted? @WiredSpast
> 
> I do but I'm mid exams so don't have a lot of time to review PR's

Fair enough. Hope exams are going well? :)

**Pull request merged**